### PR TITLE
Handle nil build spec when reading creation time

### PR DESCRIPTION
**Pull request by tomkennedy513**

### CODE REVIEW

**Review by tylerphelan — APPROVED** [review]
great catch!!

is a quick test worth it?

**Comment by tomkennedy513** [comment]
> great catch!!
> 
> 
> 
> is a quick test worth it?

Ya I'll write one now

File: pkg/apis/build/v1alpha2/image_builds_test.go [hunk 1]
```diff
@@ -349,6 +349,24 @@ func testImageBuilds(t *testing.T, when spec.G, it spec.S) {
 			assert.Equal(t, image.Spec.Notary, build.Spec.Notary)
 			assert.Equal(t, image.Spec.Cosign, build.Spec.Cosign)
 		})
+
+		it("sets the creation time when present", func() {
```
Comment (tylerphelan) [inline pkg/apis/build/v1alpha2/image_builds_test.go:353]: oh snap assumed we at least had this one

**Review by tylerphelan — APPROVED** [review]
great!

**Comment by chenbh** [comment]
If you don't mind a little scope creep, do you mind adding documentation for `Image.build.creationTime`? I had to go back to  #1032 to figure out what this field does and how to configure it.

**Review by matthewmcnew — APPROVED** [review]

**Review by chenbh — APPROVED** [review]

**Pull request merged**